connext_cmake_module/cmake/Modules/FindConnext.cmake

```diff
@@ -242,21 +242,19 @@ if(NOT _NDDSHOME STREQUAL "")
 
   if(WIN32)
     set(Connext_DEFINITIONS "RTI_WIN32" "NDDS_DLL_VARIABLE")
-    # This will be a .bat file and it will be on the PATH.
-    set(Connext_DDSGEN "rtiddsgen.bat")
-    find_program(_found "rtiddsgen_server.bat")
-    if(_found)
-      set(Connext_DDSGEN_SERVER "rtiddsgen_server.bat")
-    endif()
+    # Setting absolute path so they can be passed around regardless of the PATH env variable
+    set(Connext_DDSGEN "${Connext_HOME}\\bin\\rtiddsgen.bat")
+    set(_ddsgen_server "${Connext_HOME}\\bin\\rtiddsgen_server.bat")
   else()
     set(Connext_DEFINITIONS "RTI_LINUX" "RTI_UNIX")
     set(Connext_DDSGEN "${Connext_HOME}/bin/rtiddsgen")
-    if(NOT EXISTS "${Connext_DDSGEN}")
-      message(FATAL_ERROR "Could not find executable 'rtiddsgen'")
-    endif()
-    if(EXISTS "${Connext_HOME}/bin/rtiddsgen_server")
-      set(Connext_DDSGEN_SERVER "${Connext_HOME}/bin/rtiddsgen_server")
-    endif()
+    set(_ddsgen_server "${Connext_HOME}/bin/rtiddsgen_server")
+  endif()
+  if(NOT EXISTS "${Connext_DDSGEN}")
+    message(FATAL_ERROR "Could not find executable '${Connext_DDSGEN}'")
```

Comment:
Since the logic doesn't "find" the executable anymore I think the error message should be updated to something like:

> RTI code generator doesn't exist: ${Connext_DDSGEN}

Reply:
Fixed message